OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -186,6 +194,15 @@ private void SpeedRunButton_OnClick()
             _justEnteredGame = true;
             GameObject.Find(RESUME_BUTTON_NAME).GetComponent<SubmitActionLoadScene>().Submit();
         }
+        
+        private void ResetRunButton_OnClick()
+        {
+            _modEnabled = true;
```

Comment:
I think this line is probably not necessary; we only set this to false when we go to the title screen.

Reply:
gotcha